options/base_options.py

```diff
@@ -689,7 +689,7 @@ def initialize(self, parser):
             default=[[0]],
             type=pairs_of_ints,
             nargs="+",
-            help="mask offset (in pixels) to allow generation of a bigger object in domain B (for semantic loss) for domain A, format : width (x),height (y) for each class or only one size if square",
+            help="mask offset (in pixels) to allow generation of a bigger object in domain B (for semantic loss) for domain A, format : 'width (x),height (y)' for each class or only one size if square, e.g. '125, 55 100, 100' for 2 classes",
```

Comment:
> e.g. '125, 55 100, 100' for 2 classes

The blank spaces after the commas are not necessary, and I find them a bit confusing. 
I would write `e.g. '125,55 100,100' for 2 classes` to make the separation between classes clear